include/sway/tree/root.h

```diff
@@ -21,7 +21,7 @@ struct sway_root {
 
 	struct wlr_texture *debug_tree;
 
-	struct wl_list outputs; // sway_output::link
+	struct wl_list all_outputs; // sway_output::link
```

Comment:
Can you add a comment here clarifying what kind of outputs are included here? Then you don't need to change the variable name at all.